Dockerfile

```diff
@@ -0,0 +1,85 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest AS build
+
+# golang 1.14.12 as it's the only version available on yum at time of writing.
+ENV GOLANG_VERSION=1.14.12
+ENV GOLANGCI_VERSION=v1.32.2
+
+ENV TNF_DIR=/usr/tnf
+ENV TNF_SRC_DIR=${TNF_DIR}/tnf-src
+ENV TNF_BIN_DIR=${TNF_DIR}/test-network-function
+
+ENV TEMP_DIR=/tmp
+# Most recent version of 4.4 available at time of writing.
+ENV OC_SRC_ARCHIVE=openshift-clients-4.4.0-202006211643.p0.tar.gz
+ENV OC_SRC_URL=https://github.com/openshift/oc/archive/${OC_SRC_ARCHIVE}
+ENV OC_SRC_DIR=${TEMP_DIR}/oc-client-src
+
+# Install dependencies
+RUN yum install -y golang-${GOLANG_VERSION} jq make git
+
+# Build oc from source
+ADD ${OC_SRC_URL} ${TEMP_DIR}
+RUN mkdir ${OC_SRC_DIR} && \ 
+	tar -xf ${TEMP_DIR}/${OC_SRC_ARCHIVE} -C ${OC_SRC_DIR} --strip-components=1 && \
+	cd ${OC_SRC_DIR} && \
+	make oc && \
+	mv ./oc /usr/bin/
+
+# golangci-lint
+RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b /usr/bin ${GOLANGCI_VERSION}
+
+# Add go binary directory to $PATH
+ENV PATH="/root/go/bin:${PATH}"
+
+# This step is identical to `make install-tools` from the Makefile, but there
+# are version issues if the command is run from within the `/tnf` directory.
+# TODO: resolve the version issues to make it work with line ~50
```

Comment:
What do you mean?  Can you please share the error you receive?

This is indicative of version as `go get` cannot pin a particular release tag.  It would indicate that either:
1) We are pulling in an older version elsewhere.
2) We should vendor the tools instead of dynamically grabbing the most recent release tag.

Reply:
addressed in #105